user-docs/latest/users-roles-permissions/introduction-to-users-roles-permissions.md

```diff
@@ -0,0 +1,9 @@
+# About users, roles & permissions
+
+::: warning 🚧 This section of the user guide is a work in progress. Stay tuned!
+<br>
+:::
+
+Some features of the admin panel, as well as the content managed with Strapi itself, are ruled by a system of permissions. These permissions can be assigned to roles, which are associated with the users who have access to the admin panel, the administrators. But it is also possible to grant permissions more publicly, to give access to content to the end-users of your Strapi application.
+
+Depending on what users and their roles and permissions you want to manage, you should either use the Role Based Access Control (RBAC) feature, or the Users & Permissions plugin. Both are accessible from *General > Settings* in the main navigation of the admin panel.
```

Comment:
You added an image in the intro of the Content-Type Builder plugin.
We can maybe add one here too?